src/content/blog/whats-new-october.mdx

```diff
@@ -0,0 +1,133 @@
+---
+title: "What's new in Astro - October"
+description: "Another busy month in Astro! From Hacktoberfest to releasing Page Partials. Read up on everything that happened this month in the Astro universe."
+publishDate: "October 31, 2023"
+authors:
+  - elian
+coverImage: "/src/content/blog/_images/astro-3/blog-hero-post-header.webp"
+socialImage: "/src/content/blog/_images/astro-3/blog-social.webp"
+lang: "en"
+---
+
+## Astro releases
```

Comment:
This *might* feel a little much, especially right off the top? I'm honestly not sure. We can maybe assume that people reading our blog posts have read these posts, too, so maybe this doesn't need to be as prominent? Would love other's ideas here.